test/dotnet-remove-proj.Tests/dotnet-remove-proj.Tests.csproj

```diff
@@ -1,7 +1,4 @@
-﻿<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" />
-  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
```

Comment:
I'd leave this dir.props in. Tests can reference other common resources and it will be useful to not have to add back the dir.props reference.